content/influxdb/cloud-dedicated/reference/internals/storage-engine.md

```diff
@@ -150,6 +170,18 @@ increasing the available CPU) is the most effective scaling strategy for the Com
 Horizontal scaling increases compaction throughput, but not as efficiently as
 vertical scaling.
 
+### Garbage collector
+
+The Garbage collector runs background jobs that evict expired or deleted data
+and reclaim space in both the [Catalog](#catalog) and the
+[Object store](#object-store).
```

Comment:
Perhaps it is worth mentioning here that it also includes other "garbage" files created as a result of the compaction service.

Reply:
```suggestion
The Garbage collector runs background jobs that evict expired or deleted data,
remove obsolete compaction files, and reclaim space in both the [Catalog](#catalog) and the
[Object store](#object-store).
```